smartsim/settings/base.py

```diff
@@ -586,6 +587,7 @@ def __init__(
         self._batch_cmd = batch_cmd
         self.batch_args = batch_args or {}
         self._preamble: t.List[str] = []
+        self._nodes: t.Optional[int] = None
```

Comment:
Forgot to mention, in my last review: This `attr` is no longer needed or used on the base class and should be free to drop!